desktop-widgets/preferences/preferences_defaults.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>561</width>
-    <height>558</height>
+    <width>588</width>
+    <height>600</height>
```

Comment:
this always worries me - I assume it's just automatic tool output, but we used to struggle a lot with the preferences window suddenly not fitting in our design goal screen size of 1024x600 and people not being able to change preferences

Reply:
Yes - that was my fault. Apparently I played with the window-size in Qt Creator. I wonder: Couldn't the window-size be calculated from the preferred sizes of the widgets...?

I removed the gratuitous `leftMargin`, etc. changes. It's really rude of Qt Creator to mangle these things.